src/Data/Bin/Induction.agda

```diff
@@ -0,0 +1,76 @@
+------------------------------------------------------------------------
+-- The Agda standard library
+--
+-- Arranging a well-founded recursion by _<_ on Bin.
+------------------------------------------------------------------------
+
+{-# OPTIONS --without-K --safe #-}
+
+module Data.Bin.Induction where
+
+open import Data.Bin.Base
+open import Data.Bin.Properties
+open import Data.Bin.Ordering
+open import Data.List using (List; []; _∷_; [_])
+open import Data.Nat as ℕ using (ℕ)
+import Induction.Nat
+open import Relation.Binary.PropositionalEquality using (subst)
+open import Relation.Nullary using (yes; no)
+open import Relation.Nullary.Negation using (contradiction)
+open import Induction.WellFounded using (Acc; module Inverse-image; module Subrelation)
+open Acc using (acc)
+
+Acc< =  Acc _<_
+
+<-acc-ℕ :  {n : ℕ} → Acc ℕ._<_ n
+-- This means that _<_ is well-founded on ℕ
+-- ("each number in ℕ is accessible from 0 by _<_").
+
+<-acc-ℕ {n} = Induction.Nat.<-wellFounded n
+
```

Comment:
This should at most be a `private` alias. We should not be introducing
multiple names for the same thing.

Reply:
This place is totally improved. See further notes.